airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/AirbyteDebeziumHandler.java

```diff
@@ -107,6 +107,7 @@ public AutoCloseableIterator<AirbyteMessage> getIncrementalIterators(final Confi
                                                                        final Instant emittedAt,
                                                                        final boolean addDbNameToState) {
     LOGGER.info("Using CDC: {}", true);
+    LOGGER.info("Using DBZ version: {}", "2.4.0");
```

Comment:
can we _not_ hardcode this? What happens if you try `DebeziumEngine.class.getPackage().getImplementationVersion()` instead ?